uiplib/scheduler.py

```diff
@@ -105,7 +111,9 @@ def changeCycle(self):
                 time.sleep(0.1)
 
     def setStartTime(self, time):
```

Comment:
I'm making the style and docs guide in wiki section. Will post the issue and link it just after.